<launch>

   <include file="$(find planning_environment)/test/testing_infrastructure.launch"/>

   <node pkg="planning_environment" type="environment_server" output="screen" name="environment_server"/>       
  
   <node pkg="ompl_ros_interface" type="ompl_ros" name="ompl_planning" output="screen">
     <rosparam command="load" file="$(find ompl_ros_interface)/test/ompl_planning.yaml" />
   </node>

   <node pkg="trajectory_filter_server" name="trajectory_filter" type="trajectory_filter_server" output="screen">
     <rosparam command="load" file="$(find move_arm)/test/config/joint_limits.yaml"/>
     <rosparam command="load" file="$(find move_arm)/test/config/filters.yaml"/>
   </node>

   <node pkg="pr2_arm_kinematics_constraint_aware" type="pr2_arm_kinematics_constraint_aware" name="pr2_right_arm_kinematics" output="screen">    
   	 <!-- IK parameters -->
   	 <param name="tip_name" value="r_wrist_roll_link" />     
   	 <param name="root_name" value="torso_lift_link" />  

	 <param name="group" type="string" value="right_arm" />
   </node>

   <test test-name="test_simple_trajectory_pipeline" pkg="move_arm" type="test_simple_trajectory_pipeline" time-limit="10000.0"/>
  
</launch>
